tests/unit/s2n_client_supported_groups_extension_test.c

```diff
@@ -259,13 +259,19 @@ int main()
             };
 
             for (size_t i = 0; i < NUM_MISMATCH_PQ_TEST_POLICY_OVERRIDES; i++) {
-                EXPECT_SUCCESS(s2n_enable_tls13_in_test());
+                DEFER_CLEANUP(struct s2n_config *config = s2n_config_new(),
+                        s2n_config_ptr_free);
+                EXPECT_NOT_NULL(config);
+                EXPECT_SUCCESS(s2n_config_set_cipher_preferences(config, "default_tls13"));
```

Comment:
Is this change necessary, or can `s2n_enable_tls13_in_test()` just be removed here with no other changes? It seems like the security policy is overridden anyway.

Reply:
Today `s2n_enable_tls13_in_test` always forces the use of the "default_tls13" policy.

After we add default support for TLS 1.3, it will be possible to use a FIPS TLS 1.3 policy by default, even after calling `s2n_enable_tls13_in_test()`. A FIPS policy breaks this test, hence why I am pinning the test.

Reply:
Ah I missed `{ NULL, NULL }` in the test cases, so the security policy isn't always being swapped to something specific. I see why this change is needed now.

I'm not opposed to removing `s2n_enable_tls13_in_test()`. Though in this case `s2n_disable_tls13_in_test()` should also be removed from the end of this loop.

Reply:
I actually ended up keeping `s2n_enable_tls13_in_test()` because its modifying global state across an entire test file and its better to keep the old assumptions